region/lowernorfair/east.json

```diff
@@ -530,34 +582,53 @@
           "to": [
             {
               "id": 2,
-              "requires": [
-                {"or": [
-                  "heatProof",
-                  "canHeatRun"
-                ]},
-                {"or": [
-                  {"and": [
+              "strats": [
+                {
+                  "name": "Screw",
+                  "notable": false,
+                  "requires": [
+                    "canNavigateHeatRooms",
                     "ScrewAttack",
                     {"heatFrames": 550}
-                  ]},
-                  {"and": [
-                    "canUsePowerBombs",
-                    {"heatFrames": 800},
-                    {"acidFrames": 50},
-                    {"ammo": {
-                      "type": "PowerBomb",
-                      "count": 1
-                    }}
-                  ]},
-                  {"and": [
+                  ]
+                },
+                {
+                  "name": "Space Screw",
+                  "notable": false,
+                  "requires": [
+                    "canNavigateHeatRooms",
+                    "ScrewAttack",
+                    {"heatFrames": 400}
+                  ]
+                },
+                {
+                  "name": "Shinespark",
```

Comment:
Space+blue suit uses half the energy of a spark :eyes:

Reply:
Good idea. I'll add that in. It's also lees heat frames than Space Screw because we know we have some momentum, which Space Screw can't assume.